packages/core/src/routes/admin-user.test.ts

```diff
@@ -280,6 +280,18 @@ describe('adminUserRoutes', () => {
     expect(updateUserById).not.toHaveBeenCalled();
   });
 
+  it('PATCH /users/:userId/is-suspended', async () => {
```

Comment:
Don't we need a batch API?

Reply:
We don't have any batch API yet, maybe in the future...